pkg/deploy/controller/deployerpod/controller.go

```diff
@@ -28,7 +30,7 @@ type DeployerPodController struct {
 // transientError is an error which will be retried indefinitely.
 type transientError string
 
-func (e transientError) Error() string { return "transient error handling deployer pod: " + string(e) }
+func (e transientError) Error() string { return "test error handling deployer pod: " + string(e) }
```

Comment:
Just a find and replace bug.

On Tue, Feb 2, 2016 at 4:49 AM, Michail Kargakis <notifications@github.com>
wrote:

> In pkg/deploy/controller/deployerpod/controller.go
> <https://github.com/openshift/origin/pull/6930#discussion_r51545525>:
>
> > @@ -28,7 +30,7 @@ type DeployerPodController struct {
> >  // transientError is an error which will be retried indefinitely.
> >  type transientError string
> >
> > -func (e transientError) Error() string { return "transient error handling deployer pod: " + string(e) }
> > +func (e transientError) Error() string { return "test error handling deployer pod: " + string(e) }
>
> transient errors may happen in any kind of deployment (test or not), I
> think this should stay as is (a "transient error" is much more clear than a
> "test error")
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/6930/files#r51545525>.
>